### ISSUE TITLE
trigger ENFILE error with live reload

### ISSUE THREAD

**Issue by latel** [thgh/rollup-plugin-livereload#10]
the file that triggers ENFILE error varies every time.

```
node_modules/.bin/rollup -c build/rollup.dev.js -w
bundling...
http://localhost:10001 -> /Users/kevinnwang/codes/logline
LiveReload enabled
bundled in 805ms. Watching for changes...
Error: watch /Users/kevinnwang/codes/logline/node_modules/mocha/node_modules/glob/test/globstar-match.js ENFILE
    at _errnoException (util.js:1003:13)
    at FSWatcher.start (fs.js:1397:19)
    at Object.fs.watch (fs.js:1423:11)
    at createFsWatchInstance (/Users/kevinnwang/codes/logline/node_modules/chokidar/lib/nodefs-handler.js:37:15)
    at setFsWatchListener (/Users/kevinnwang/codes/logline/node_modules/chokidar/lib/nodefs-handler.js:80:15)
    at FSWatcher.NodeFsHandler._watchWithNodeFs (/Users/kevinnwang/codes/logline/node_modules/chokidar/lib/nodefs-handler.js:228:14)
    at FSWatcher.NodeFsHandler._handleFile (/Users/kevinnwang/codes/logline/node_modules/chokidar/lib/nodefs-handler.js:255:21)
    at FSWatcher.<anonymous> (/Users/kevinnwang/codes/logline/node_modules/chokidar/lib/nodefs-handler.js:473:21)
    at FSReqWrap.oncomplete (fs.js:167:5)
```

**Comment by latel** [comment]
my config file:

```javasvript
// Rollup plugins
import * as path from 'path';
import babel from 'rollup-plugin-babel';
import uglify from 'rollup-plugin-uglify';
import license from 'rollup-plugin-license';
import serve from 'rollup-plugin-serve';
import livereload from 'rollup-plugin-livereload';
import * as jsonfile from 'jsonfile';

const pkg = jsonfile.readFileSync('./package.json');

export default {
    entry: 'src/' + pkg.name + '.js',
    dest: 'dist/' + pkg.name + (process.env.NODE_ENV === 'production' ? '.min' : '') + '.js',
    format: 'umd',
    moduleName: pkg.name.replace(/^\w/, starter => starter.toUpperCase()),
    sourceMap: process.env.NODE_ENV === 'production',
    plugins: [
        babel({
            exclude: 'node_modules/**'
        }),
        (process.env.NODE_ENV === 'production' && uglify()),
        license({
            banner: {
                file: path.join(__dirname, 'src', 'BANNER')
            },
            thirdParty: {
                output: path.join(__dirname, '../dist', 'dependencies.txt'),
                includePrivate: true
            }
        }),
        serve({
            contentBase: './',
            open: true
        }),
        livereload()
    ]
};

```

**Comment by thgh** [comment]
I don't know what that error means. Maybe the dist folder doesn't exist at that point?
Have you tried to pass a folder to watch?
```
livereload('dist')
```

**Comment by thgh** [comment]
This is probably fixed in v1.2.0, will close for now. Always welcome to update with repo that reproduces the issue.
